docs/product/custom-dashboards.md

```diff
@@ -5,74 +5,92 @@ sidebar_position: 6
 
 # Custom Dashboards
 
-In addition to our pre-built dashboards for topics like Crashes, Logs, and Network Requests, each project can create Custom Dashboards.  In these dashboards, you can create new analyses.  Visualize a time series of Crash data segmented by app-version, or get a ranking of Logs filtered by a specific property.
+In addition to our pre-built dashboards for topics like Crashes, Logs, and Network Requests, each project can create Custom Dashboards. In these dashboards, you can create new analyses. Visualize a time series of Crash data segmented by app-version, or get a ranking of Logs filtered by a specific property.
 
 To get started, click on "Add new widget" in the menu by your dashboard name.
-<img src={require('@site/static/images/Spans widgets > 01 add menu.png').default} alt="create new widget" />
+<img src={require('@site/static/images/custom_dashboards/Add_Widget_Menu.png').default} style={{ width: '75%', height: '75%' }} alt="Create new widget" />
 
-## Spans (Traces)
+## Creating a Chart
+You can create multiple types of visualizations. A table of which visualizations are supported is shown below:
+
+| Chart Type | Crashes | Network | Logs | Issues | Moments | Sessions | Spans | Historical | Multi-query |
+| ---------- | ------- | ------- | ---- | ------ | ------- | -------- | ----- | ---------- | ----------- |
+| Line | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Table | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :x:
+| Bar | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :x: |
+| Pie | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :x: |
 
-Create graphs monitoring Spans performance.  You can filter and group by name, outcome, duration, and any attributes you have set on the Span.
+## Spans (Traces)
 
-First, select the Spans category when making a new Widget:
-<img src={require('@site/static/images/widget_data_types_new.png').default} alt="Spans as a Widget option" />
+Create graphs monitoring Spans performance. You can filter and group by name, outcome, duration, and any attributes you have set on the Span.
 
-Then, choose the metric you wish to aggregate.  For Spans, Embrace supports both counts and sums of duration.
-<img src={require('@site/static/images/Spans widgets > 03 pick metric.png').default} alt="Span metric options" />
+- First, select the visualization type you want to see and a source for the first query (more on multi-query combined time series below).
```

Comment:
Leave out the reference to the first query if the visualization is open-ended. If we want this particular example to have multiple queries then we can mention the first.